<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>467</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QPushButton" name="button4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#D8BFD8;
	border: 2px solid gray;
	padding: 5px;
}

QPushButton:pressed{
	background-color:#8B008B;
	border: 2px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="button5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#D8BFD8;
	border: 2px solid gray;
	padding: 5px;
}

QPushButton:pressed{
	background-color:#8B008B;
	border: 2px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="button8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#D8BFD8;
	border: 2px solid gray;
	padding: 5px;
}

QPushButton:pressed{
	background-color:#8B008B;
	border: 2px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="button3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#D8BFD8;
	border: 2px solid gray;
	padding: 5px;
}

QPushButton:pressed{
	background-color:#8B008B;
	border: 2px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="button6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#D8BFD8;
	border: 2px solid gray;
	padding: 5px;
}

QPushButton:pressed{
	background-color:#8B008B;
	border: 2px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="button7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#D8BFD8;
	border: 2px solid gray;
	padding: 5px;
}

QPushButton:pressed{
	background-color:#8B008B;
	border: 2px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="buttonPoint">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#E6E6FA;
	border: 2px solid gray;
	padding: 5px;
}

QPushButton:pressed{
	background-color:#8B008B;
	border: 2px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="button2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#D8BFD8;
	border: 2px solid gray;
	padding: 5px;
}

QPushButton:pressed{
	background-color:#8B008B;
	border: 2px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="button1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#D8BFD8;
	border: 2px solid gray;
	padding: 5px;
}

QPushButton:pressed{
	background-color:#8B008B;
	border: 2px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="button9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#D8BFD8;
	border: 2px solid gray;
	padding: 5px;
}

QPushButton:pressed{
	background-color:#8B008B;
	border: 2px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="buttonAC">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#E6E6FA;
	border: 2px solid gray;
	padding: 5px;
}

QPushButton:pressed{
	background-color:#8B008B;
	border: 2px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="buttonPM">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#E6E6FA;
	border: 2px solid gray;
	padding: 5px;
}

QPushButton:pressed{
	background-color:#8B008B;
	border: 2px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="buttonPercent">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#E6E6FA;
	border: 2px solid gray;
	padding: 5px;
}

QPushButton:pressed{
	background-color:#8B008B;
	border: 2px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="buttonDivision">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#E6E6FA;
	border: 2px solid gray;
	padding: 5px;
}

QPushButton:pressed{
	background-color:#8B008B;
	border: 2px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>÷</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="buttonMultip">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#E6E6FA;
	border: 2px solid gray;
	padding: 5px;
}

QPushButton:pressed{
	background-color:#8B008B;
	border: 2px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="buttonAddition">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#E6E6FA;
	border: 2px solid gray;
	padding: 5px;
}

QPushButton:pressed{
	background-color:#8B008B;
	border: 2px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="buttonSubstract">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#E6E6FA;
	border: 2px solid gray;
	padding: 5px;
}

QPushButton:pressed{
	background-color:#8B008B;
	border: 2px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="buttonEquals">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#E6E6FA;
	border: 2px solid gray;
	padding: 5px;
}

QPushButton:pressed{
	background-color:#8B008B;
	border: 2px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QPushButton" name="button0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#D8BFD8;
	border: 2px solid gray;
	padding: 5px;
}

QPushButton:pressed{
	background-color:#8B008B;
	border: 2px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QLineEdit" name="calculatorDisplay">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color:#FFFFFF;
	border:1px solid gray;
	color: #000000;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>465</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>qtCalculator</string>
    </property>
    <addaction name="actionExit"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
